content/sensu-go/5.15/reference/etcdreplicators.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Etcd replicators"
+description: "Etcd replicators allow you to manage RBAC resources in one place and mirror the changes to follower clusters. Read the reference to set up RBAC policy replication between Sensu clusters."
+weight: 10
+version: "5.15"
+product: "Sensu Go"
+menu:
+  sensu-go-5.15:
+    parent: reference
+---
+
+- [Create a replicator](#create-a-replicator)
+- [Delete a replicator](#delete-a-replicator)
+- [Replicator configuration](#replicator-configuration)
+- [etcd-replicators specification](#etcdreplicators-specification)
+- [Example etcd-replicators resources](#example-etcdreplicators-resources)
+
+**LICENSED TIER**: Unlock the etcd-replicators datatype in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][1].
```

Comment:
With the introduction of 5.15, there are no longer paywalled features, including etcd-replicators, so this line can be removed.